packages/encryption/src/lib/schemas/LPACC_ATOM.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$id": "https://litprotocol.com/schemas/LPACC_EVM_ATOM.json",
```

Comment:
This will not work as a schema hosting platform. can we instead host schemas in a separate repo and link explicitly to that through github.

Reply:
we can also use this `$id` for versioning eg. `LPACC_EVM_ATOM_V1.json`